app/resources/admin.html.template

```diff
@@ -116,12 +153,15 @@
     //add the languages dropdown
     var languages_col = row.insertCell(cur_col++);
     var languages = document.createElement('select');
-    languages.id = 'language_selection_' + iteration;
+    languages.id = 'repo_titles_language_selection_' + iteration;
+    languages.onchange = function() {update_language_selected(iteration);check_language_selected(iteration);};
```

Comment:
Please break lines to fit lines under 80 chars:

languages.onchange = function() {
    update_language_selected(iteration);
    check_language_selected(iteration);
};